internal/dao/sqlx/account_dao.go

```diff
@@ -10,6 +10,7 @@ import (
 )
 
 const (
+	createAccount             = `INSERT INTO accounts (account_number, org_id) VALUES ($1, $2) ON CONFLICT DO UPDATE RETURNING id`
```

Comment:
Can you drop ON CONFLICT DO UPDATE? It must be followed with a statement what to do actually. Apologies I misled you in the ticket, my bad.